tests/test_api.py

```diff
@@ -6,14 +6,21 @@
 from decimal import Decimal
 
 from toml.decoder import InlineTableDict
-
+from toml.decoder import loads
```

Comment:
nit: this import is unnecessary. `toml` is imported above, so you can use `toml.loads`